nengo/utils/progress.py

```diff
@@ -782,11 +781,7 @@ def get_default_progressbar():
     if pbar.lower() == "none":
         return NoProgressBar()
 
-    try:
-        return _load_class(pbar)()
-    except Exception as e:
-        warnings.warn("Could not load progress bar: %s" % e)
-        return NoProgressBar()
```

Comment:
Do we want to fail if we can't load it? A warning seems better to me.

Reply:
Eh, if a user gets to this point they're doing something pretty specific, so I think they would not be upset to be told about it (a warning is easy to ignore / miss). I think 0% of users have seen this warning.